BetterMenus/Utils.cs

```diff
@@ -74,6 +73,29 @@ public static MenuBuilder AddControlButton(this MenuBuilder builder, string name
                     }));
         }
 
+        public static MenuBuilder AddBackButtonToBMenu(this MenuBuilder builder, Menu menuRef, out UnityEngine.UI.MenuButton backButton)
```

Comment:
why is this method named `AddBackButtonToBMenu` and not  `AddBackButtonToMenu` ? possible typo?